specify_manual/command_design/show_annotattion/io.md

```
@@ -56,12 +57,16 @@ It also makes it possible to switch between `config/default.toml` and `config/si
+- `summary`: Overall statistics (count, types, etc.).
+- `list`: File-based summary list (default).
+- `group`: Grouped by attributes (Layer, Type, etc.).
+- `detail`: Full annotation details.
 
-### JSON Output
+### Format Options
 
-A structured representation suitable for machine processing or the `report-ui`.
\ No newline at end of file
+- `text`: Human-readable text (Standard Output/Error).
+- `json`: Machine-readable structured representation.
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)